public/manifest.json

```diff
@@ -1,21 +1,18 @@
 {
-  "short_name": "React App",
-  "name": "Create React App Sample",
+  "short_name": "QuickSwap",
+  "name": "QuickSwap Interface",
   "icons": [
     {
-      "src": "favicon.ico",
-      "sizes": "64x64 32x32 24x24 16x16",
-      "type": "image/x-icon"
-    },
-    {
-      "src": "logo192.png",
+      "src": "./logo_circle.png",
+      "sizes": "50x150",
       "type": "image/png",
-      "sizes": "192x192"
+      "purpose": "any maskable"
```

Comment:
what is this?

Reply:
Manifest was loading the invalid images and it returns 404 error. I updated manifest.json to fix the error.